Microsoft.Toolkit.Uwp.UI.Controls.Layout/ContentSizer/ContentSizer.cs

```diff
@@ -12,14 +15,8 @@ namespace Microsoft.Toolkit.Uwp.UI.Controls
     /// The <see cref="ContentSizer"/> is a control which can be used to resize any element, usually its parent. If you are using a <see cref="Grid"/>, use <see cref="GridSplitter"/> instead.
     /// </summary>
     [ContentProperty(Name = nameof(Content))]
```

Comment:
Oh, I think we can share the attribute in the base class?

Reply:
Confirmed.  The attribute is now only applied to the base class.